src/test/java/jssc/bootpath/ManualBootLibraryPathFailedTest.java

```diff
@@ -10,17 +10,12 @@
  * Tests if a valid <code>jssc.boot.library.path</code> which does NOT contain a native library
  * will predictably fail.  This test can be run regardless of whether or not a native binary was
  * created during the build process.
- *
- * TODO: This MUST be in its own class to run in a separate JVM (https://stackoverflow.com/questions/68657855)
- * - JUnit does NOT currently offer JVM unloading between methods.
- * - maven-surefire-plugin DOES offer JVM unloading between classes using <code>reuseForks=false</code>
- * - Unloading is needed due to NativeLoader.loadLibrary(...) calls System.loadLibrary(...) which is static
```

Comment:
I'd not remove this comment from affected test. Executing all tests in Intellij (as opposed to with maven) still fails because of the issue described here, so it should stay where the problem is.

Reply:
It wasn't remove, it was moved, but thanks for the attention to detail.  I've moved it back and added more context.